src/RulesDeploy.ts

```diff
@@ -135,7 +140,7 @@ RulesDeploy.prototype = {
       );
       return Promise.resolve();
     });
-  },
-};
+  }
+}
 
-module.exports = RulesDeploy;
+export = RulesDeploy;
```

Comment:
with the `export` keyword in the `class` declaration, this line won't be necessary.

Reply:
done